src/main/java/com/jvm_bloggers/frontend/public_area/all_issues/AllIssuesPageBackingBean.java

```diff
@@ -33,7 +34,11 @@ public AllIssuesPageBackingBean(NewsletterIssueForListingQuery query) {
                     .sorted(Comparator.comparing(NewsletterIssueForListing::getIssueNumber))
                     .reverse()
             );
-        return TreeMap.ofAll(issuesGroupedByYearMonth.toJavaMap());
+        return TreeMap.ofEntries(
+            Comparator.reverseOrder(),
+            issuesGroupedByYearMonth
+                .map(Tuple2::new)
+                .toArray());
```

Comment:
Can we move this mapping to the temp variable `issuesGroupedByYearMonth` just to have simple map creation over here?